client/my-sites/marketing/tools/index.tsx

```diff
@@ -3,30 +3,40 @@
  */
 import { connect } from 'react-redux';
 import page from 'page';
-import PropTypes from 'prop-types';
 import React, { Fragment, FunctionComponent } from 'react';
 import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
-import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { FEATURE_GOOGLE_MY_BUSINESS, PLAN_BUSINESS } from 'lib/plans/constants';
+import GoogleMyBusinessLocation from 'my-sites/google-my-business/location';
+import getGoogleMyBusinessConnectedLocation from 'state/selectors/get-google-my-business-connected-location';
+import { getSelectedSiteSlug, getSelectedSiteId } from 'state/ui/selectors';
 import MarketingToolsFeature from './feature';
+import MarketingToolsFeatureButtonsWithPlanGate from './feature-buttons-with-plan-gate';
 import MarketingToolsHeader from './header';
 import { marketingSharingButtons, marketingTraffic } from 'my-sites/marketing/paths';
+import QueryKeyringConnections from 'components/data/query-keyring-connections';
+import QueryKeyringServices from 'components/data/query-keyring-services';
+import QuerySiteKeyrings from 'components/data/query-site-keyrings';
```

Comment:
It may be better to keep concerns separated by creating a new `GoogleMyBusinessFeature` card that will take care of fetching the right data, and connecting the component. Otherwise `MarketingTools` is going to be a mess very soon 😄.